src/porepy/numerics/nonlinear/nonlinear_solvers.py

```diff
@@ -147,34 +147,39 @@ def newton_step() -> None:
                     )
                     newton_step()
                     solver_progressbar.update(n=1)
-                    solver_progressbar.set_postfix_str(f"Error {error_norm}")
+                    solver_progressbar.set_postfix_str(f"Inc. {error_inc}")
```

Comment:
Isn't "Inc." too ambiguous?

Reply:
Changed to "Increment" now. Better? As this is going to be printed to the screen, I would be cautious with too long names.